inst/pages/14_alpha_diversity.qmd

```diff
@@ -108,16 +108,84 @@ plotColData(tse,
             "SampleType", 
             colour_by = "Final_Barcode") +
     theme(axis.text.x = element_text(angle=45,hjust=1)) + 
-  labs(y=expression(Richness[Observed]))
```

Comment:
The "labs" notation is nowadays the recommended way to do this. The ylab might be deprecated later. Could you switch back to "labs"?